scripts/build.sh

```diff
@@ -36,11 +36,10 @@ if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]
   set +e
   git add .
   set -e
-  git status
-  # H/T: https://github.com/dhermes
```

Comment:
I believe this is just a hat-tip to dhermes because we stole this code (written by him) from gcloud-python.